tools/components/internal/table.ts

```diff
@@ -3,7 +3,7 @@ import { SortOrder } from '@tools/types'; // Sort order enum
 
 // Function type of getSortOptions - Disable eslint preventing `object` from being used
 // eslint-disable-next-line
-type GetSortOptions = (sortBy: SortingRule<object>[]) => ReadonlyArray<[string, SortOrder]>;
+type GetSortOptions = (sortBy: SortingRule<object>[]) => Array<[string, SortOrder]>;
```

Comment:
Just wondering, why did we change this to `Array`?

Reply:
this was a side effect of removing the readonly from the other type